react/features/base/participants/middleware.js

```diff
@@ -37,30 +39,15 @@ MiddlewareRegistry.register(store => next => action => {
         break;
 
     case KICK_PARTICIPANT:
-        if (typeof APP !== 'undefined') {
-            APP.UI.emitEvent(UIEvents.USER_KICKED, action.id);
-        }
+        conference.kickParticipant(action.id);
+
         break;
 
-    case MUTE_REMOTE_PARTICIPANT:
-        if (typeof APP !== 'undefined') {
-            APP.UI.messageHandler.openTwoButtonDialog({
-                titleKey: 'dialog.muteParticipantTitle',
-                msgString:
-                    '<div data-i18n="dialog.muteParticipantBody"></div>',
-                leftButtonKey: 'dialog.muteParticipantButton',
-                dontShowAgain: {
-                    id: 'dontShowMuteParticipantDialog',
-                    textKey: 'dialog.doNotShowMessageAgain',
-                    checked: true,
-                    buttonValues: [ true ]
-                },
-                submitFunction: () => {
-                    APP.UI.emitEvent(UIEvents.REMOTE_AUDIO_MUTED, action.id);
-                }
-            });
-        }
+    case MUTE_REMOTE_PARTICIPANT: {
```

Comment:
Silly style nit: The rest of the switch doesn't use curly braces so maybe this line shouldn't either. Curly braces would be needed for scoping variable declarations.